compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -1143,17 +1136,21 @@ class OMR_EXTENSIBLE CodeGenerator
    //
    // Used to track the internal control flow depth level while compiling.
    // Updated the CodeGenerator.hpp in x86, s390, and PPC so they reference this common code.
+
+   private:
+
    int32_t _internalControlFlowNestingDepth;
    int32_t _internalControlFlowSafeNestingDepth;
-   TR::Instruction *_instructionAtEndInternalControlFlow;
+
+   public:
+
    int32_t internalControlFlowNestingDepth() {return _internalControlFlowNestingDepth;}
    int32_t internalControlFlowSafeNestingDepth() { return _internalControlFlowSafeNestingDepth; }
    void incInternalControlFlowNestingDepth() {_internalControlFlowNestingDepth++;}
    void decInternalControlFlowNestingDepth() {_internalControlFlowNestingDepth--;}
    bool insideInternalControlFlow() {return (_internalControlFlowNestingDepth > _internalControlFlowSafeNestingDepth);}
    void setInternalControlFlowNestingDepth(int32_t depth) { _internalControlFlowNestingDepth = depth; }
    void setInternalControlFlowSafeNestingDepth(int32_t safeDepth) { _internalControlFlowSafeNestingDepth = safeDepth; }
```

Comment:
Not sure if this is the right PR for these, but more naming nitpicks:

`internalControlFlowNestingDepth()` can be changed to `getInternalControlFlowNestingDepth` and `internalControlFlowSafeNestingDepth()` can be changed to `getInternalControlFlowSafeNestingDepth()`. This way the API naming convention will be consisted with how the setters are named. 

We can also change `insideInternalControlFlow()` to `isInsideInternalControlFlow()`.

Reply:
I think this is an issue outside the scope of this PR as these are common APIs used across other codegens as well. We can however open an issue for this if you think it merits additional work?

Reply:
Agreed. I've opened an issue here: https://github.com/eclipse/omr/issues/3373